### PR TITLE
Remove regions and enable commands from console and some related fixes

### PR DESCRIPTION
**Pull request by Czompi**
## Remove regions and enable commands from console and some related fixes

- [x] Remove regions from MainCommandModule (72dc2a8a5fbe465e0bf8edbd9187bc87991846d5)
*I might know who added them* 😅
- [x] Grant all permissions to the console (b1737ca6aeb41ceef1bddfcea6f1373eefc5c0fe).
- [x] Enabled most of the commands to be executed from console (05439e7898ea2ccdc1311b9ac86439bd3bd4e1c2).

Ps.: Sorry for the lot of merges in this PR.

### CODE REVIEW

**Comment by Seb-stian** [comment]
> Sorry for the lot of merges in this PR.

We can squash it to keep git history clean. No problem.

File: Obsidian/Commands/MainCommandModule.cs [hunk 1]
```diff
@@ -15,13 +15,14 @@ namespace Obsidian.Commands
 {
     public class MainCommandModule
     {
-        #region help
         private const int CommandsPerPage = 15;
         [Command("help", "commands")]
         [CommandInfo("Lists available commands.", "/help [<page>]")]
+        [IssuerScope(CommandIssuers.Any)]
```
Comment (Craftplacer) [inline Obsidian/Commands/MainCommandModule.cs]: This is common theme I've seen by people writing code here recently -- you're all too *explicit* with attributes.

Using no `IssuerScopeAttribute` should always default to all scopes. Please change it that when no attribute is provided it falls back to `Any` and delete the "unnecessary" lines.

Anyway, the rest of the PR looks fine though.

**Review by Craftplacer — CHANGES_REQUESTED** [review]

**Review by Craftplacer — APPROVED** [review]

File: Obsidian/Commands/Framework/CommandHandler.cs [hunk 2]
```diff
@@ -75,7 +75,7 @@ public void RegisterSingleCommand(Action method, PluginContainer plugin, Type t)
             var checks = m.GetCustomAttributes<BaseExecutionCheckAttribute>();
 
             var info = m.GetCustomAttribute<CommandInfoAttribute>();
-            var issuers = m.GetCustomAttribute<IssuerScopeAttribute>()?.Issuers ?? CommandIssuers.Client;
+            var issuers = m.GetCustomAttribute<IssuerScopeAttribute>()?.Issuers ?? CommandIssuers.Any;
```
Comment (Czompi) [inline Obsidian/Commands/Framework/CommandHandler.cs]: May I add an internal `DefaultIssuerScope` field and use that instead of `CommandIssuers.Any`?

Comment (Craftplacer) [reply Obsidian/Commands/Framework/CommandHandler.cs]: A constant value can be used to ensure consistency, sure.

**Review by Naamloos — APPROVED** [review]
Looks fine to me, great job

**Pull request merged**